testnets/airchainstestnet/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "airchainstestnet",
+  "assets": [
+    {
+      "description": "The native staking token of Airchains Testnet.",
+      "denom_units": [
+        {
+          "denom": "amf",
+          "exponent": 0
+        },
+        {
+          "denom": "amf",
```

Comment:
Ambiguous denoms, does amf mean 0 exponent or 6 exponent? They must be different.

Reply:
Regarding the ambiguous `denom` values:

Since we have this configuration set up in the chain itself, that's why it is written this way. The chain's minimal denom and denom are both set to be `amf`, hence the duplication. If this creates an issue, can we just add the minimal denom here and leave out the normal denom? Would that work for the requirements?

Reply:
It is problematic to have both be the same, and there is a strict requirement that they be different.
It is quite rare that there is no second denom, so I'd just like to ask confirmation that the asset, when represented by the token symbol AMF will never be divisible on any interface?
In most cases, the spelling of non-base denoms are arbitrary, so if an AMF is meant to be divisible into 1 million units, then I'd recommend adding the exp:6 denom back in, but changing the string to something like "AMF" (upper case).

Reply:
I don't think this will affect transactions, because all commands to a chain generally only use the base denom ('amf'). The display denom (AMF) is just used to display the asset on application interfaces.
However, merely being included in the denom_untis array isn't enough to tell interfaces to use AMF over amf; there's one last change expected here, and that is to change the "display" value to "AMF" (to uppercase, to match the new denom unit).

Reply:
Done